packages/dx-react-scheduler/docs/reference/drag-drop-provider.md

```diff
@@ -0,0 +1,77 @@
+# DragDropProvider Plugin Reference
+
+A plugin that enables users to edit appointments via drag-and-drop.
+
+## Import
+
+Use the following statement to import a plugin with embedded theme components:
+
+```js
+import { DragDropProvider } from '@devexpress/dx-react-scheduler-material-ui';
+```
+
+If you want to use custom components, you can import the themeless plugin:
+
+```js
+import { DragDropProvider } from '@devexpress/dx-react-scheduler';
+```
+
+## User Reference
+
+### Dependencies
+
+- [EditingState](editing-state.md)
+- [Appointments](week-view.md)
+- [DayView](day-view.md) [Optional]
+- [WeekView](week-view.md) [Optional]
+- [MonthView](month-view.md) [Optional]
+- [AllDayPanel](all-day-panel.md) [Optional]
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+allowDrag | (appointmentData: [AppointmentModel](./scheduler.md#appointmentmodel)) => boolean | () => true | A function that specifies draggable appointments.
+draftAppointmentComponent | ComponentType&lt;[DragDropProvider.DraftAppointmentProps](#dragdropproviderdraftappointmentprops)&gt; | | A component that renders the appointment being dragged.
+sourceAppointmentComponent | ComponentType&lt;[DragDropProvider.SourceAppointmentProps](#dragdropprovidersourceappointmentprops)&gt; | | A component that renders the copy of the appointment being dragged in the former location.
```

Comment:
```suggestion
sourceAppointmentComponent | ComponentType&lt;[DragDropProvider.SourceAppointmentProps](#dragdropprovidersourceappointmentprops)&gt; | | A component that renders a copy of the appointment being dragged in the former location.
```

Reply:
in the former location?

Reply:
While you are dragging an appointment, a copy of it is displayed where the appointment has been. Once you drop the appointment, the copy disappears

Reply:
```suggestion
sourceAppointmentComponent | ComponentType&lt;[DragDropProvider.SourceAppointmentProps](#dragdropprovidersourceappointmentprops)&gt; | | A component that renders a copy of the appointment being dragged in its previous location.
```